library/javascript/components/slider.js

```diff
@@ -123,16 +123,25 @@ document.addEventListener('DOMContentLoaded', function (e) {
 
     const leftArrowElement = document.createElement('img')
     leftArrowElement.className = 'arrow'
-    leftArrowElement.setAttribute('src', leftArrow)
+    leftArrowElement.setAttribute('data-icon', 'carouselleft')
+    leftArrowElement.classList.add('ui')
+    leftArrowElement.classList.add('icon')
+    leftArrowElement.classList.add('prev')
+    leftArrowElement.classList.add('icon_image')
```

Comment:
Is the class icon_image already there? Was it that since it was not include the arrows used yo work bad?

Reply:
all the class names that I have added are present in the other dark mode arrows too. But the actual classes that are responsible for the dark mode feature are "icon" and "data-icon"